src/components/Pipelines/ExecutionAreaPipelineAgent.vue

```diff
@@ -0,0 +1,372 @@
+<template>
+  <div class="row force-scroll mt-4">
+    <div v-for="(item2,index) of associatedAgents" :key="item2.id" class="col-5"
+      :class="{'p-0': true}"
+      style="position: relative;">
+      <div :class="{'invisible': Object.keys(item2).length === 0}">
+        <div class="info-box-background float-left w-65"
+          style="position: relative; left: 0px; top: -1px;">
+        </div>
+        <div :class="{ 'invisible': index+1 === associatedAgents.length || Object.keys(associatedAgents[index+1]).length === 0}"
+          class="mt-3 margin-center abs-center border-top w-35"
+          style="color:black!important;border: 1px solid; float:left"> </div>
+          <div v-if="index+1 !== associatedAgents.length" :class="{ 'invisible': index+1 === associatedAgents.length || Object.keys(associatedAgents[index+1]).length === 0 }" class="mt-3 black-circle">  </div>
```

Comment:
This is very hard to understand
```
"index+1 !== associatedAgents.length" 
:class="{ 'invisible': index+1 === associatedAgents.length || Object.keys(associatedAgents[index+1]).length === 0 }"
```
Can we move that to a method and call the method instead ??
And use better conversion name